src/views/teacherregistration/l10n.json

```diff
@@ -36,5 +36,6 @@
     "teacherRegistration.howUseScratch": "How do you plan to use Scratch at your organization?",
     "teacherRegistration.emailStepTitle": "Email Address",
     "teacherRegistration.emailStepDescription": "We will send you a confirmation email that will allow you to access your Scratch Teacher Account.",
-    "teacherRegistration.validationEmailMatch": "The emails do not match"
+    "teacherRegistration.validationEmailMatch": "The emails do not match",
+    "teacherRegistration.validationAge": "Sorry, teachers must be at least 13 years old"
```

Comment:
@kaschm I added in this language if the user happens to have been born in the 13th year of their life, but has not yet gotten to their birth month. Does this seem alright to you content-wise?